provisions/roles/db/tasks/main.yml

```diff
@@ -1,18 +1,32 @@
 ---
-- name: Ensure cccp data dir exists
+- name: Check if NFS backup directory exists on node
+  sudo: yes
+  stat:
+    path: '{{db_backup_nfs_path}}'
+  register: nfs_dir_mounted
+  when: not test
+  tags: db
+
+- name: Fail if NFS share is not mounted on node for db backup
+  sudo: yes
+  fail:
+    msg: "NFS share is not mounted on node for db backup."
+  when:
+    - not test
```

Comment:
I did not get what does this job check. how is it coming to conclusion of nfs is not mounted

Reply:
This is to ensure the NFS directory is present on the node for cron job. And we don't need cron job in CI, so this will not run in CI, i.e. when `test=False` in inventory.